test/controllers/notifications.controller.test.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach, afterEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Things we need to stub
+const ReturnsPeriodService = require('../../app/services/notifications/setup/returns-period.service.js')
+
+// For running our service
+const { init } = require('../../app/server.js')
+
+describe('Notifications Setup controller', () => {
+  const basePath = '/notifications/setup'
+
+  let options
+  let server
+
+  beforeEach(async () => {
+    // Create server before each test
+    server = await init()
+
+    // We silence any calls to server.logger.error made in the plugin to try and keep the test output as clean as
+    // possible
+    Sinon.stub(server.logger, 'error')
+
+    // We silence sending a notification to our Errbit instance using Airbrake
+    Sinon.stub(server.app.airbrake, 'notify').resolvesThis()
+  })
```

Comment:
Small NIT. Moving the `await init()` call into a `before()` block to avoid `MaxListenersExceededWarning` as I've done in [Fix MaxListenersExceededWarning in tests](https://github.com/DEFRA/water-abstraction-system/pull/1519).

```suggestion
  // Create server before running the tests
  before(async () => {
    server = await init()
  })

  beforeEach(async () => {
    // We silence any calls to server.logger.error made in the plugin to try and keep the test output as clean as
    // possible
    Sinon.stub(server.logger, 'error')

    // We silence sending a notification to our Errbit instance using Airbrake
    Sinon.stub(server.app.airbrake, 'notify').resolvesThis()
  })
```

Note - You'll need to add `before` into the require() at the top after accepting/doing this suggestion.